decidim-debates/app/services/decidim/debates/diff_renderer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Debates
+    class DiffRenderer < BaseDiffRenderer
+      private
+
+      # Lists which attributes will be diffable and how they should be rendered.
+      def attribute_types
+        {
+          title: :string,
```

Comment:
The translatable fields can have a `:i18n` value so when rendering it in the diff view is more user-friendly.

```suggestion
          title: :i18n,
```